### PR TITLE
luci-app-spoofer: add package

### PR DESCRIPTION
**Pull request by kenkeys**
Signed-off-by: Ken Keys <kkeys@caida.org>

### CODE REVIEW

File: applications/luci-app-spoofer/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (C) 2018 The Regents of the University of California
```
Comment (karlp) [inline applications/luci-app-spoofer/Makefile]: Do you actually have a copyright assignment on file with UC?

File: applications/luci-app-spoofer/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=Spoofer LuCI interface
+LUCI_DEPENDS:=+luci-mod-admin-full +spoofer
+
+PKG_MAINTAINER:=Ken Keys <kkeys@caida.org>
+PKG_LICENSE:=GPL-3.0
```
Comment (karlp) [inline applications/luci-app-spoofer/Makefile]: Is this correct for the luci app?  What's the intention on how that interacts with other luci apps? Note that this is independent of the license for the spoofer application itself.

File: applications/luci-app-spoofer/Makefile (hunk 1 above)
Comment (kenkeys) [reply applications/luci-app-spoofer/Makefile]: Yes, I do.  (I'm the same person who added the spoofer package to openwrt, and released the upstream spoofer package it's built on.)

File: applications/luci-app-spoofer/Makefile (hunk 2 above)
Comment (kenkeys) [reply applications/luci-app-spoofer/Makefile]: The only intention was to license this as open source; I wasn't aware this had any effect on interaction with other luci apps.  Is there an explanation somewhere?  What would you suggest?  I'm sure any reasonable open source license would be acceptable on my end.

Comment (jow-) [reply applications/luci-app-spoofer/Makefile]: @kenkeys - according to http://www.apache.org/licenses/GPL-compatibility.html there seems to be an issue with mixing GPL v3.0 software with Apache 2.0 (which most of LuCI is licensed as).

Maybe simply relicense to Apache 2.0, then there shouldn't be a problem.

**Comment by sukhyi** [comment]
`/usr/lib/lua/luci/dispatcher.lua:584: Invalid controller file found
The file '/usr/lib/lua/luci/controller/spoofer.lua' contains an invalid module line.
Please verify whether the module name is set to 'luci.controller.spoofer' - It must correspond to the file path!
stack traceback:
	[C]: in function 'assert'
	/usr/lib/lua/luci/dispatcher.lua:584: in function 'createindex'
	/usr/lib/lua/luci/dispatcher.lua:610: in function 'createtree'
	/usr/lib/lua/luci/dispatcher.lua:279: in function 'dispatch'
	/usr/lib/lua/luci/dispatcher.lua:144: in function </usr/lib/lua/luci/dispatcher.lua:143>`

**Comment by feckert** [comment]
@kenkeys could you please stash (fixup) your commits into one? This is a new package so we do not need the changes history till it gets merged upstream.

**Comment by feckert** [comment]
No reaction from pullrequest creator so I will close this.

**Pull request closed without merging**